<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2000</width>
    <height>950</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow { 
background-color: #6291b6 ;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="bas1" native="true">
    <property name="geometry">
     <rect>
      <x>815</x>
      <y>290</y>
      <width>200</width>
      <height>180</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QCustomplot{
background-color:#6291b6


}</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="bas2" native="true">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>290</y>
      <width>200</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCustomPlot{
border-style: solid;
  border-width: 4px;
  border-radius: 50px;
  border-color: dark blue;
	padding: 10px;

}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled/>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>45</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TELEMETRI</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="temp" native="true">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>490</y>
      <width>200</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="height2" native="true">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>90</y>
      <width>200</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="lspeed" native="true">
    <property name="geometry">
     <rect>
      <x>815</x>
      <y>690</y>
      <width>200</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>450</y>
      <width>221</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans</family>
      <pointsize>-1</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	
	color: #fff;
	font-size: 20px;
	font-weight: bold;
border-style: solid;
 

	padding: 10px;


}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="qml.qrc">
      <normaloff>:/başlatj.png</normaloff>
      <normalon>:/durdurj.png</normalon>:/başlatj.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCustomPlot" name="height1" native="true">
    <property name="geometry">
     <rect>
      <x>815</x>
      <y>90</y>
      <width>200</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="QQuickWidget" name="quickWidget">
    <property name="geometry">
     <rect>
      <x>1275</x>
      <y>90</y>
      <width>260</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="resizeMode">
     <enum>QQuickWidget::ResizeMode::SizeRootObjectToView</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="rgb">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>211</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans</family>
      <pointsize>-1</pointsize>
      <italic>true</italic>
      <bold>true</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	
	color: #fff;
	font-size: 20px;
	font-weight: bold;
	border-style: solid;



	padding: 15px;

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="qml.qrc">
      <normaloff>:/filtrelej.png</normaloff>:/filtrelej.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QQuickWidget" name="quickWidget_2">
    <property name="geometry">
     <rect>
      <x>1275</x>
      <y>740</y>
      <width>400</width>
      <height>280</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QquickWidget{
border-style: solid;
  border-width: 4px;
  border-radius: 50px;
  border-color: dark blue;
	padding: 10px;

}</string>
    </property>
    <property name="resizeMode">
     <enum>QQuickWidget::ResizeMode::SizeRootObjectToView</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>800</width>
      <height>371</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget {
background-color: #052b47;
border-style: solid;
  border-width: 4px;
  border-radius: 50px;
  border-color: #5AED72;
	padding: 10px;

}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="movement">
     <enum>QListView::Movement::Snap</enum>
    </property>
    <property name="flow">
     <enum>QListView::Flow::TopToBottom</enum>
    </property>
    <property name="resizeMode">
     <enum>QListView::ResizeMode::Fixed</enum>
    </property>
    <property name="viewMode">
     <enum>QListView::ViewMode::ListMode</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2211</width>
      <height>1081</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="qml.qrc">:/background1.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="qml.qrc">:/itutayf0.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>431</width>
      <height>61</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>125</red>
          <green>211</green>
          <blue>138</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>125</red>
          <green>211</green>
          <blue>138</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled/>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>68</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;ITU TAYF 1H&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>80</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active/>
      <inactive/>
      <disabled/>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>3D MODEL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>678</y>
      <width>371</width>
      <height>101</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active/>
      <inactive/>
      <disabled/>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>GPS HARİTA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>375</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Canlı Yayın</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>1600</x>
      <y>130</y>
      <width>78</width>
      <height>50</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active/>
      <inactive/>
      <disabled/>
     </palette>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_2">
    <property name="geometry">
     <rect>
      <x>1600</x>
      <y>187</y>
      <width>78</width>
      <height>50</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active/>
      <inactive/>
      <disabled/>
     </palette>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_3">
    <property name="geometry">
     <rect>
      <x>1600</x>
      <y>245</y>
      <width>78</width>
      <height>50</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active/>
      <inactive/>
      <disabled/>
     </palette>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>110</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active/>
      <inactive/>
      <disabled/>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pitch</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>165</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active/>
      <inactive/>
      <disabled/>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Roll</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>225</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active/>
      <inactive/>
      <disabled/>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Yaw</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="iot" native="true">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>690</y>
      <width>200</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="Irtifa_Farki" native="true">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>490</y>
      <width>200</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="Voltage" native="true">
    <property name="geometry">
     <rect>
      <x>815</x>
      <y>490</y>
      <width>200</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ayril">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	
	color: #fff;
	font-size: 20px;
	font-weight: bold;
border-style: solid;
 

	padding: 10px;


}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="qml.qrc">
      <normaloff>:/ayrılj.png</normaloff>:/ayrılj.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>320</width>
      <height>300</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>720</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>5</red>
          <green>43</green>
          <blue>71</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>5</red>
          <green>43</green>
          <blue>71</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>5</red>
          <green>43</green>
          <blue>71</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>5</red>
          <green>43</green>
          <blue>71</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>5</red>
          <green>43</green>
          <blue>71</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>5</red>
          <green>43</green>
          <blue>71</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>5</red>
          <green>43</green>
          <blue>71</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>5</red>
          <green>43</green>
          <blue>71</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>5</red>
          <green>43</green>
          <blue>71</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>5</red>
          <green>43</green>
          <blue>71</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>45</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {

color: #052b47;
}</string>
    </property>
    <property name="text">
     <string>İrtifa Farkı</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>1274</x>
      <y>30</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled/>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>HATA KODU</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>770</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLCDNumber {
border: 2px solid white; background-color: #052b47; color: white;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="kod1">
    <property name="geometry">
     <rect>
      <x>1440</x>
      <y>25</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="kod2">
    <property name="geometry">
     <rect>
      <x>1485</x>
      <y>25</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="kod3">
    <property name="geometry">
     <rect>
      <x>1530</x>
      <y>25</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="kod4">
    <property name="geometry">
     <rect>
      <x>1575</x>
      <y>25</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="kod5">
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>25</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>560</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>29</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #052b47;
border:  4px solid white;;
border-style: solid;
  border-width: 4px;
  border-radius: 20px;
  border-color: #5AED72;
	padding: 10px;
color: white;
}</string>
    </property>
    <property name="text">
     <string>8B2R</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <zorder>label_3</zorder>
   <zorder>bas1</zorder>
   <zorder>bas2</zorder>
   <zorder>label</zorder>
   <zorder>temp</zorder>
   <zorder>height2</zorder>
   <zorder>lspeed</zorder>
   <zorder>pushButton</zorder>
   <zorder>height1</zorder>
   <zorder>quickWidget</zorder>
   <zorder>rgb</zorder>
   <zorder>quickWidget_2</zorder>
   <zorder>listWidget</zorder>
   <zorder>logo</zorder>
   <zorder>label_2</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>lcdNumber</zorder>
   <zorder>lcdNumber_2</zorder>
   <zorder>lcdNumber_3</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>iot</zorder>
   <zorder>Irtifa_Farki</zorder>
   <zorder>Voltage</zorder>
   <zorder>ayril</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>lcdNumber_4</zorder>
   <zorder>kod1</zorder>
   <zorder>kod2</zorder>
   <zorder>kod3</zorder>
   <zorder>kod4</zorder>
   <zorder>kod5</zorder>
   <zorder>lineEdit</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qml.qrc"/>
 </resources>
 <connections/>
</ui>
